CMSgov/design-system
# [WNMGDS-986]Set up linting

## Summary
- Using Husky and lint-staged to lint all staged files before committing 

## How to test
- Make a linting issue in a JSX, TSX or SCSS file and stage it for commit and ensure that it gets either autofixed or flagged for manual fix before allowing the commit
